code/core/src/manager/components/sidebar/SidebarBottom.tsx

```diff
@@ -206,6 +229,9 @@ export const SidebarBottom = ({ isDevelopment }: { isDevelopment?: boolean }) =>
       errorCount={errorCount}
       warningCount={warningCount}
       isDevelopment={isDevelopment}
+      testProviderStates={testProviderStates}
+      testProviderInterfaces={testProviderInterfaces}
```

Comment:
I think this name is confusing. What does interfaces mean in this context? Isn't this just a list of registered test providers?

Reply:
good point. I was trying to make the distinction between states in the test provider store, and test providers that were registered in the manager UI. But `registeredTestProviders` sounds better to me.